doc/guide_3rd_party.md

```diff
@@ -142,16 +143,70 @@ Use a GSRN number from the response described in [Get Metering Points](#get-mete
 
 If the metering point owner does not already have a wallet, a new wallet will be created as part of the request. For both production and consumption certificates a contract is needed.
```

Comment:
Hvad betyder det? at man skal have en wallet for consumption og en for production, eller bare en enkelt wallet?

Reply:
Just that no matter what you need a wallet and it will be created for you.